chatterbot_corpus/data/english/trivia.yml

```
@@ -26,4 +26,6 @@ conversations:
 - - Dolphins use a sense, similar to sonar, to determine the location and shape of
     nearby items.
   - Echolocation
-
+- - Account related change affected areas?
+  -Customer, Treasury , Securities
```

```suggestion
  - Customer, Treasury , Securities
```
You are missing a space here